src/views/QA/QA.tsx

```diff
@@ -1,53 +1,49 @@
 import React, { useState, useEffect } from 'react';
 import Dropdown, { Option } from 'react-dropdown';
+import { withRouter, RouteComponentProps } from 'react-router';
 import qs from 'query-string';
 
 //Interfaces
-import IQuestion from '../../interfaces/IQuestion';
-import ISubject from '../../interfaces/ISubject';
-import IGrade from '../../interfaces/IGrade';
+import { IQuestion, ISubject } from '../../interfaces';
 
 //Services
-import {
-  getQuestionList,
-  getSubjectList,
-  getGradeList,
-} from '../../services/api-service';
+import { getQuestionList, getSubjectList } from '../../services/api-service';
 
-//Components
-import QAList from '../../ui/components/QAList';
+//Sections
+import { SectionHelper, SectionQAList, SectionPagination } from './Sections';
+import gradeList from '../../grades';
 
 interface IProps {
   location;
-  history;
 }
 
-export const QA = (props: IProps) => {
+export const QA = (props: IProps & RouteComponentProps) => {
+  const { history, location } = props;
   //Fetched questions
   const [questions, setQuestions] = useState([] as IQuestion[]);
   //Dropdown alternatives
   const [subjects, setSubjects] = useState([] as ISubject[]);
-  const [grades, setGrades] = useState([] as IGrade[]);
 
   //Parsed query parameter string
-  const values = qs.parse(props.location.search);
+  const values = qs.parse(location.search);
 
   //Returns option to be filled out based on query params
   const getDefaultOptions = type => {
-    return { value: values[type] || '' };
+    return { value: values[type] || type === 'page' ? '0' : '' };
```

Comment:
kan skrive type === 'page' && '0' for å korte ned litt

Reply:
Sett, endrer :)

Reply:
Eller nei, den må settes som tom streng for å unngå undefined.